src/main/java/ca/corefacility/bioinformatics/irida/ria/web/sessionAttrs/Cart.java

```diff
@@ -88,4 +88,13 @@ public int removeProject(Long projectId) {
 		return Sets.newHashSet(this.values());
 	}
 
+	/**
+	 * Find if a sample is currently in the cart.  If it is get the project that it belongs to.
+	 *
+	 * @param sampleId identifier for a sample sample to check the cart for.
```

Comment:
Extra sample in @param text

Reply:
Updated in 25828a13e932bd65ed5dc10eaede867a2af9e306